### PR TITLE
Check for changelog entry in PR bodies

### PR DESCRIPTION
**Pull request by flip1995**
cc #4031 but now on the auto and try branches. (https://github.com/rust-lang/rust-clippy/issues/3955#issuecomment-502230021)

changelog: none

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -92,6 +93,20 @@ matrix:
     - os: windows
 
 script:
+  - |
+    if [ "$TRAVIS_BRANCH" == "auto" ] || [ "$TRAVIS_BRANCH" == "try" ]; then
+      pr=$(echo $TRAVIS_COMMIT_MESSAGE | grep -o "#[0-9]*" | head -1 | sed 's/^#//g')
+      output=$(curl -H "Authorization: token $GITHUB_API_TOKEN" -s "https://api.github.com/repos/rust-lang/rust-clippy/pulls/$pr" | \
+        python -c "import sys, json; print(json.load(sys.stdin)['body'])" | \
+        grep "changelog: " | \
```
Comment (matthiaskrgr) [inline .travis.yml]: should this be tightened to be "^changelog: "? or would that be too restrictive?

Comment (flip1995) [reply .travis.yml]: Yeah, I wondered the same. E.g. #4210 would fail, because the changelog is italic.

Comment (flip1995) [reply .travis.yml]: Done 👍

**Comment by matthiaskrgr** [comment]
@bors r+
thanks!

**Pull request merged**